conduit/v1/toproto/schema.go

```diff
@@ -0,0 +1,55 @@
+// Copyright © 2024 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package toproto
+
+import (
+	"fmt"
+
+	"github.com/conduitio/conduit-commons/schema"
+	conduitv1 "github.com/conduitio/conduit-connector-protocol/proto/conduit/v1"
+)
+
+func GetResponse(inst schema.Instance) *conduitv1.GetSchemaResponse {
```

Comment:
One request around naming - the other `fromproto` and `toproto` packages in this repo follow a strict naming scheme, where the name of the function is the same as the type name it's converting. So if we introduce `conduit.GetSchemaResponse` as I propose in the other comment, then this would be changed to:

```suggestion
func GetSchemaResponse(in conduit.GetSchemaResponse) *conduitv1.GetSchemaResponse {
```